config/locales/nl_intro.yml

```diff
@@ -26,8 +26,8 @@ nl:
     multi_year_charts:
       title: Multi-year charts
       description: |
-        See how a 2050 scenario can become reality by exploring 2023, 2030, and
-        2040.
+        Ontdek hoe een toekomstscenario voor 2050 realiteit kan worden door
+        zowel 2023, 2030 als 2040 te verkennen.
```

Comment:
Ik denk dat de huidige verwoording teveel suggereert dat het 'realistisch' is en daarmee de voorspellende waarde toeneemt. 

De essentie is wat mij betreft eerder dat je hiermee "kunt verkennen wat de mogelijk *paden* zijn naar een 2050 eindbeeld" en "wat dat zou kunnen betekenen voor de tussentijdse doelen die je wil nastreven".

Reply:
For the home page I've changed the translations into:

> Verken de mogelijke transitiepaden naar een 2050 toekomstbeeld en ontdek wat dit zou kunnen betekenen voor de tussentijdse doelstellingen in 2023, 2030 en 2040. 

> Explore the possible transition paths to a scenario for 2050 and discover the actions and consequences for aspired goals in 2023, 2030 and 2040.

For the myc page I've changed the translations into:

> Verken de mogelijke transitiepaden naar een 2050 toekomstbeeld en ontdek wat dit zou kunnen betekenen voor de tussentijdse doelstellingen voor de belangrijkste zichtjaren (2023, 2030 en 2040). Begin door een van je opgeslagen toekomstscenario's voor 2050 te selecteren.

> Explore the possible transition paths to a scenario for 2050 and discover the actions and consequences for aspired goals in 2023, 2030 and 2040. Start by selecting one of your saved 2050 scenarios below.

Reply:
Nice!